feat: Add cloudwatch and security group outputs

adamcrews opened the pull request:
## what

This adds the outputs for the optionally created cloudwatch logs and the security group.

## why

In my use case, this information is useful to pass on to other modules, thus adding the outputs helps expose that information.

This also supersedes PR https://github.com/cloudposse/terraform-aws-ec2-client-vpn/pull/116
and closes issue https://github.com/cloudposse/terraform-aws-ec2-client-vpn/issues/91

Gowiem commented on outputs.tf:
```diff
@@ -34,3 +19,38 @@ output "full_client_configuration" {
+
+output "security_group_name" {
+  value       = local.security_group_enabled ? module.vpn_security_group.name : null
+  description = "The ID of the security group associated with the Client VPN endpoint."
```
```suggestion
  description = "The name of the security group associated with the Client VPN endpoint."
```

adamcrews replied:
Thanks for the catch.  Fixed.

Gowiem reviewed (changes requested):
Small changes. Running tests to confirm all is good 👍

Gowiem commented:
/terratest

Gowiem commented:
@adamcrews this is annoying, but can you add descriptions to all of the outputs in `examples/complete`? Appreciate it.

```
Test: check if terraform outputs have descriptions
File: output-descriptions.bats
---------------------------------
client_configuration is missing a description
vpn_endpoint_arn is missing a description
vpn_endpoint_dns_name is missing a description
vpn_endpoint_id is missing a description
---------------------------------

not ok 1 check if terraform outputs have descriptions
# (in test file /test/terraform/output-descriptions.bats, line 16)
#   `[ -z "$output" ]' failed
+ status=fail
+ [[ fail == \p\a\s\s ]]
+ pass=false
+ printf '\n\nEnd of testing in /__w/terraform-aws-ec2-client-vpn/terraform-aws-ec2-client-vpn/examples/complete\n\n\n'
+ [[ false == \t\r\u\e ]]


End of testing in /__w/terraform-aws-ec2-client-vpn/terraform-aws-ec2-client-vpn/examples/complete
```

adamcrews commented:
> @adamcrews this is annoying, but can you add descriptions to all of the outputs in `examples/complete`? Appreciate it.

No worries.  It looks like these failures pre-date my changes, but I added descriptions to the existing outputs, and added all outputs to the example/complete code.

Gowiem reviewed (approved):
LGTM

Gowiem commented:
/terratest
